pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -4411,6 +4411,10 @@ span.gwt-RadioButton label {
   margin-top: -3px;
 }
 
+#contentCleanerPanel tbody tr td table tbody tr td table tbody tr td .gwt-TextBox {

```

Comment:
Can't understand this limitation 50px.. Looks like box-sizing: border-box; fixes the issue.